qcengine/tests/test_procedures.py

```diff
@@ -19,8 +19,11 @@ def input_data():
 
 
 @using("psi4")
-@using("geometric")
-def test_geometric_psi4(input_data):
+@pytest.mark.parametrize(
+    "optizer",
+    [pytest.param("geometric", marks=[using("geometric")]), pytest.param("optking", marks=[using("optking")])],
```

Comment:
No need for marks to be a list here. Can we write out `optimizer`?

Reply:
Fixed